app/src/app/components/header/AccountHeaderDropdown.tsx

```diff
@@ -27,14 +27,15 @@ export default function HeaderDropDown() {
       <DropdownMenuTrigger asChild>
         <Button variant="ghost" className="relative h-10 w-10 rounded full">
           <Avatar className="h-8 w-8">
-            <AvatarImage src="https://github.com/shadcn.png" alt="shadcn" />
+            <AvatarImage src="/img/avatar.webp" alt="avatar" />
             <AvatarFallback>LA</AvatarFallback>
           </Avatar>
```

Comment:
As I said, I think we should lose the avatar, we're not going to be fetching that info from github, and it doesn't exist in montagu.